jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -730,9 +731,31 @@ static Instant getCreationTime(String configuredCreationTime, ProjectProperties
           return Instant.now();
 
         default:
-          return DateTimeFormatter.ISO_DATE_TIME.parse(configuredCreationTime, Instant::from);
+          DateTimeFormatter formatter =
+              new DateTimeFormatterBuilder()
+                  .parseCaseInsensitive()
```

Comment:
Is this intended? The JDK code you linked doesn't start with this.

Reply:
Yeah, I was just playing around with things, mustve left it in there.